### PR TITLE
Move first scan after the AppSec devtools plugin first init

### PR DESCRIPTION
**Pull request by tamasmak**
- This ensures that the first vulnerability scan during the startup is run after AppSec devtools plugin is setup
- Improves the scan event listener registration debug message

### CODE REVIEW

File: appsec-kit-backend/src/main/java/com/vaadin/appsec/backend/AppSecService.java [hunk 1]
```diff
@@ -221,9 +221,10 @@ private void cancelScheduledScan() {
      *            the listener
      * @return a registration object that can be used to remove the listener
      */
-    public Registration addScanEventListener(AppSecScanEventListener listener) {
+    public Registration addScanEventListener(AppSecScanEventListener listener,
+            String listenerName) {
```
Comment (heruan) [inline appsec-kit-backend/src/main/java/com/vaadin/appsec/backend/AppSecService.java:225]: This new parameter has the only purpose to feed a log message, feels a bit an overkill and it's also a breaking change.

If we want to identify the caller let's use the caller's logger instead, i.e. log the message right after adding the listener.

**Review by heruan — CHANGES_REQUESTED** [review]

File: appsec-kit-backend/src/main/java/com/vaadin/appsec/backend/AppSecService.java (hunk 1 above)
Comment (tamasmak) [reply appsec-kit-backend/src/main/java/com/vaadin/appsec/backend/AppSecService.java]: That makes sense. Removed the parameter and added log messages in the caller class.

File: appsec-kit-v7/src/main/java/com/vaadin/appsec/v7/ui/content/MainView.java [hunk 2]
```diff
@@ -30,6 +33,10 @@
  * Results tab content
  */
 public class MainView extends AbstractAppSecContent {
```
Comment (heruan) [inline appsec-kit-v7/src/main/java/com/vaadin/appsec/v7/ui/content/MainView.java:35]: To make the logger name meaningful, I suggest to rename this class to `AppSecView` as we have in the V24 module.

File: appsec-kit-v8/src/main/java/com/vaadin/appsec/v8/ui/content/MainView.java [hunk 3]
```diff
@@ -31,6 +34,9 @@
  */
 public class MainView extends AbstractAppSecContent {
```
Comment (heruan) [inline appsec-kit-v8/src/main/java/com/vaadin/appsec/v8/ui/content/MainView.java:35]: To make the logger name meaningful, I suggest to rename this class to `AppSecView` as we have in the V24 module.

**Review by heruan — CHANGES_REQUESTED** [review]

File: appsec-kit-v7/src/main/java/com/vaadin/appsec/v7/ui/content/MainView.java (hunk 2 above)
Comment (tamasmak) [reply appsec-kit-v7/src/main/java/com/vaadin/appsec/v7/ui/content/MainView.java]: Good point. Renamed.

File: appsec-kit-v8/src/main/java/com/vaadin/appsec/v8/ui/content/MainView.java (hunk 3 above)
Comment (tamasmak) [reply appsec-kit-v8/src/main/java/com/vaadin/appsec/v8/ui/content/MainView.java]: Good point. Renamed.

**Review by heruan — APPROVED** [review]

**Pull request merged**